packages/publisher/src/generate-packages.ts

```diff
@@ -36,7 +36,7 @@ if (require.main === module) {
   const tgz = !!yargs.argv.tgz;
   logUncaughtErrors(async () => {
     const log = loggerWithErrors()[0];
-    const options = { ...defaultLocalOptions, definitelyTypedPath: outputDirPath };
+    const options = { ...defaultLocalOptions };
```

Comment:
Hm, how did this work before? I noticed this before when debugging publishing the first time and wasn't sure why we were setting this to an output path...

Reply:
No idea! I don’t think I’ve ever run `pnpm generate` before but it was definitely broken for me